<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LJ_V7060</class>
 <widget class="QMainWindow" name="LJ_V7060">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
   </widget>
   <widget class="QMenu" name="menuCommunication">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionUSB"/>
    <addaction name="actionEntherner"/>
    <addaction name="actionCommClose"/>
    <addaction name="actionCommunicationSetting"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionGetVersion"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCommunication"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionUSB">
   <property name="text">
    <string>USB</string>
   </property>
  </action>
  <action name="actionEntherner">
   <property name="text">
    <string>Entherner</string>
   </property>
  </action>
  <action name="actionCommClose">
   <property name="text">
    <string>CommClose</string>
   </property>
  </action>
  <action name="actionGetVersion">
   <property name="text">
    <string>GetVersion</string>
   </property>
  </action>
  <action name="actionCommunicationSetting">
   <property name="text">
    <string>CommunicationSetting</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
